Trimble.Modus.Components/Controls/Button/CustomButton.xaml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8" ?>

```

Comment:
Do we need this class? If yes, can you rename the class to something more meaningful?

Reply:
Have to remove the TMButton. Once it's removed have to rename this class to TMButton. TMbutton has some reference in modals and toaster

Reply:
Can you remove that as part of this PR? Or can you create a separate ticket for that to do in the next sprint?